msteams-platform/mstdd-landing.yml

```diff
@@ -78,26 +78,26 @@ landingContent:
   linkLists:
     - linkListType: whats-new
       links:
+      - text: Receive participant events
```

Comment:
For clarity use: "Receive meeting participant events"

Reply:
Updated